Makefile

```diff
@@ -13,6 +13,7 @@ all: default
 SOURCES = \
 	src/backend.c \
 	src/backends/carbon.c \
+        src/backends/opentsdb.c \
```

Comment:
Not sure if github is deceiving me, but seems to be like you have a tabs vs spaces issue here and below in the enum.